### PR TITLE
Implement wave markers 

### PR DESCRIPTION
**Pull request by pjanik**
https://www.pivotaltracker.com/n/projects/2441242/stories/180370979

- Change header in blue bar to Sound Visualizer.
- Make the Amplitude I beam on the right of the dot at the nearest peak with the word Amplitude next to it.
- wavelength will be shown with a teal |-| horizontal I-beam. The label Wavelength with a numeric value of the wavelength in milliseconds will appear below. 
- Frequency will be shown in the upper right corner of the graph.
- Remove labeling from modulated wave.
- Hitting play will turn labeling off.

I had only a few hours today to implement that, so the code, variable names, etc., might not be very polished. But I think this gives us working markers and should be easy to move on from here.

### CODE REVIEW

**Review by Concord-Jim — COMMENTED** [review]
Looks good, overall.

However, the added/changed code in the `use-sound-wave-rendering.ts` file is a little hard to follow. Would it be possible to add some additional comments there? In particular, to clarify how the mapping from samples to pixels (for the marker lines) is done...

**Comment by pjanik** [comment]
> Looks good, overall.
> 
> However, the added/changed code in the `use-sound-wave-rendering.ts` file is a little hard to follow. Would it be possible to add some additional comments there? In particular, to clarify how the mapping from samples to pixels (for the marker lines) is done...

It's these 3 lines:
```javascript
const xScale = width / pointsCount;
const minPointIdx = currentDataPointIdx - 0.5 * pointsCount; // currentDataPointIdx is always in the middle
const xScaled = (amplitudeX - minPointIdx) * xScale - 0.5 * kLineWidth;
```
`width` is the width in pixels, `pointsCount` is all the points being drawn within `<width> px`. And `currentDataPointIdx` is the index of the point in the middle of the graph. 

I agree that the rendering code is not obvious. But it's mostly tricky because of the zooming and padding that needs to be added. So, I don't know how simple it can get. Sometimes I just take a piece of paper and follow an easy example to get what I need. Or draw shapes step by step. Also, there are now enough rendering functions to have almost all necessary transformations.

I think it'd be nice to follow that and just extract more helpers with descriptive names. And/or think about how to simplify all the rendering. I'd be happy to do it, but I won't have time to do it before I leave this weekend.
